plugin.py

```diff
@@ -16,3 +19,33 @@ class LspElmPlugin(NpmClientHandler):
     server_binary_path = os.path.join(
         server_directory, 'node_modules', '@elm-tooling', 'elm-language-server', 'out', 'index.js'
     )
+
+    @classmethod
+    def required_node_version(cls) -> str:
+        return ">=14"
+
+    def on_pre_server_command(self, params: Mapping[str, Any], done_callback: Callable[[], None]) -> bool:
+        if params['command'] == 'editor.action.showReferences':
+            arguments = cast(ShowReferencesAction, params['arguments'])
+            session = self.weaksession()
+            if not session:
+                return False
```

Comment:
is it ok to return False here and on line 35?

I do not want to handle the command if I am missing a session or a view.

Reply:
Should return True and return empty result IMO. Otherwise server we'll send `workspace/executeCommand` to the server which it won't know how to handle I suppose.